Libs/DICOM/Widgets/ctkDICOMTableView.h

```diff
@@ -109,33 +109,126 @@ class CTK_DICOM_WIDGETS_EXPORT ctkDICOMTableView : public QWidget
    * entries with the according uids are selected
    * @param uids a list of uids which should be selected
    */
-  void setQuery(const QStringList &uids = QStringList());
+  Q_INVOKABLE void setQuery(const QStringList &uids = QStringList());
 
   /**
-   * @brief Add a where condition to the usual select statement
-   * @param condition std::pair with column name and a value list
+   * @brief Add a where condition to the select statement that filters for matching values
+   * @param condition std::pair with column name and a value list in
+   *        which the column needs to have one of the values in the list
    */
   Q_INVOKABLE void addSqlWhereCondition(const std::pair<QString, QStringList>& condition);
 
   /**
-   * @brief Add a where condition to the usual select statement
-   * @param condition std::pair with column name and a value list
+   * @brief Add a where condition to the select statement that filters for matching values.
+   * @param column Column name that needs to have one of the values in the values list
+   * @param values List of values that are accepted for the given column
+   * 
+   * Same as \sa addSqlEqualWhereCondition
+   * Combines with the rest of the where conditions (\sa addSqlGreaterWhereCondition, \sa addSqlLessWhereCondition,
+   * \sa addSqlGreaterEqualWhereCondition, \sa addSqlLessEqualWhereCondition) using 'and',
+   * e.g. "column1 in ('value11','value12') and column2 in ('value21','value22') and column3 > value3 and column4 <= value4"
    */
   Q_INVOKABLE void addSqlWhereCondition(const QString column, const QStringList& values);
```

Comment:
```
  Q_INVOKABLE void addSqlWhereCondition(const QString& column, const QStringList& values);
```

I didn't specify it as a `suggestion` as the implementation should also be updated. See https://github.com/commontk/CTK/pull/955/files#diff-d5fca64865dfa90b6433baea9d318054f273a6ca2eb83325a3c055ad29037089L612